guides/source/active_record_querying.md

```diff
@@ -1277,14 +1277,20 @@ class Article < ApplicationRecord
 end
 ```
 
-This is exactly the same as defining a class method, and which you use is a matter of personal preference:
+This is almost the same as defining a class method, except for that scopes always return an `ActiveRecord::Relation` object. 
```

Comment:
**exactly** the same is not true, but the remaining part is almost the same with here:

https://github.com/rails/rails/blob/97feb4996b1c88f770101dfce6d4d3a6baf6bb33/guides/source/active_record_querying.md#L1360-L1370

Reply:
How about removing this part in favor of the info below in order to not repeat yourself?

Reply:
Yeah, it is a good idea to me.